sh/tests/bin-status_test.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+FILENAME="student/bin-status.sh"
+
+
+challenge() {
+	submitted=$(eval "$1" ; bash $FILENAME)
+	expected=$(eval "$1" ; bash solutions/bin-status.sh)
+
+	diff <(echo "$submitted") <(echo "$expected")
+}
+
+
+# True if FILE exists and is a regular file
+if [ -f ${FILENAME} ]; then
```

Comment:
i have been testing a bit and found that if you use `echo "0"` as solution, the test will pass.

Reply:
in the test i test for different cases, return 0 and return != 0

Reply:
Yes i see the test has some cases but it is returning  0
So if you put echo "0" the tests pass

Reply:
fixed, see comment for the details